### ISSUE TITLE
Account page skips H2 element

### ISSUE THREAD

**Issue by torgeirl** [devilry/devilry-django#1181]
Success Criteria 1.3.2 in WCAG 2 fails due incorrect header usage:

* `/account/` skips from `<h1>` (handled by the `cradmin-legacy-page-header-inner` class) to `<h3>` elements (handled by the `devilry-dashboard-container` class); no `<h2>` is displayed.

**Comment by jimma1** [comment]
@torgeirl Hello. There are several solutions for this task. Below comes examples for two solutions which I mean are equally good. Which one do you prefer?

**Use h2**
Replace h3 with h2:
![image](https://user-images.githubusercontent.com/10887266/130564269-fedbfe67-9f47-4436-bd9d-c999763864d3.png)


**Change h1 text and add h2**
Set the h1 to be equal the page title and add a new h2 with the current h1 text.
![image](https://user-images.githubusercontent.com/10887266/130564596-5510321b-a9ec-46a3-858b-122f45a0fea4.png)

**Comment by StianJul** [comment]
@jimma1, if I remember correctly, we have CSS size classes you could use instead.

Which means that we can keep the first example you proposed by simply replacing the `<h3>` with `<h2 class="h3">`. This would ensure the visuals are the same as before, but the hierarchical structure is correct (not skipping h2).

**Comment by jimma1** [comment]
@StianJul That was my first try, what you suggests about using class='h3' on a h2 tag. However, this does look good. And we have a lot of css for .h3. So my opinion is that it is easier to 

`Account Overview` = h2
`Name` = h2 with class="h3"
`Short name` = h3


![image](https://user-images.githubusercontent.com/10887266/130605292-f2b9944f-ffa2-4316-998b-d11ffaf437c2.png)
